pkg/piecestore/psserver/psdb/psdb_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/bmizerany/assert"
```

Comment:
We are using `"github.com/stretchr/testify/assert"`

Reply:
thnx...updated